src/components/button/buttonVariant.ts

```diff
@@ -0,0 +1,42 @@
+import { type VariantProps, cva } from 'class-variance-authority';
+
+export type BaseButtonProps = VariantProps<typeof button>;
+
+export const button = cva(['font-semibold', 'border'], {
```

Comment:
can we name it something else like  'buttonVariant' or maybe something else? what do you say @singh12345akshay  @Ankit9589599  @jpatidar-1998

Reply:
i have change name of file as cva.ts as SamyakSir suggested